docs/core/entity/select.md

```diff
@@ -0,0 +1,35 @@
+---
+title: Select Entity
+sidebar_label: Select
+---
+
+A `select` is an entity that allows the user to select an option from a list limited options provided by the integration. Derive entity platforms from [`homeassistant.components.select.SelectEntity`](https://github.com/home-assistant/home-assistant/blob/master/homeassistant/components/select/__init__.py)
```

Comment:
Should we clarify that this platform should only be used when there are no other platforms that better fit the integration feature? Eg a vacuum device should not be represented by this entity since we already have a vacuum platform.

Reply:
Pushed a clarification. I've used Lights as an example, as that is probably more common to the reader.